api/src/main/java/ai/djl/modality/cv/translator/YoloV8Translator.java

```diff
@@ -27,13 +29,17 @@
  */
 public class YoloV8Translator extends YoloV5Translator {
 
+
+    final int maxBoxes;

```

Comment:
The usage of `final` for members and parameters is generally discouraged in favor of readability.

See: https://docs.djl.ai/docs/development/development_guideline.html#coding-conventions